### PR TITLE
fix(insert-footer-button)：设计器页面设置-输入设置中，保存取消按钮顺序错误，格式化按钮样式错误，应与UI保持一直；大纲书导航中，眼睛图标，建议改为睁开就是可见，闭眼就是不可见；提示框样式修改

### PR DESCRIPTION
**Pull request by Xppp0217**
设计器页面设置-输入设置中，保存取消按钮顺序错误，格式化按钮样式错误，应与UI保持一致;
大纲树导航中，眼睛图标，建议改为睁开就是可见，闭眼就是不可见;
提示框样式修改

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/plugins/materials/src/block/BlockGroup.vue [hunk 1]
```diff
@@ -319,7 +320,10 @@ export default {
       })
     }
 
-    const handleConfirmDeleteGroup = (groupId) => {
+    const handleConfirmDeleteGroup = (group) => {
+      const groupId = group.groupId
+      const groupName = group.groupName
```
Comment (ghost) [inline packages/plugins/materials/src/block/BlockGroup.vue]: 建议使用解构语法取值 const { groupId, groupName } = group

**Review by ghost — DISMISSED** [review]
LGTM

File: packages/controller/src/useModal.jsx [hunk 2]
```diff
@@ -24,10 +24,12 @@ const confirm = ({ title, status, message, exec, cancel, showFooter = true }) =>
   })
 }
 
-const message = ({ title, status, message, exec }) => {
+const message = ({ title, status, message, exec, width = '400'}) => {
```
Comment (chilingling) [inline packages/controller/src/useModal.jsx]: 注意一下格式化

File: packages/common/component/MetaCodeEditor.vue [hunk 3]
```diff
@@ -45,12 +45,12 @@
       </div>
       <template #footer>
         <div class="btn-box">
-          <tiny-button type="danger" v-if="language === 'json' && showFormatBtn" @click="formatCode">
+          <tiny-button plain type="danger" v-if="language === 'json' && showFormatBtn" @click="formatCode">
             {{ $t('common.format') }}
           </tiny-button>
           <div>
+            <tiny-button @click="close">{{ $t('common.cancel') }}</tiny-button>
```
Comment (chilingling) [inline packages/common/component/MetaCodeEditor.vue:52]: 这里取消保存的顺序影响面积大，需要评估一下

**Review by chilingling — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**